devops/helm/crunchy-postgres/values-0c27fb-prod.yaml

```diff
@@ -29,10 +29,10 @@ instances:
     storageClassName: netapp-block-standard
   requests:
     cpu: "2"
-    memory: 2Gi
+    memory: 6Gi
```

Comment:
We also have one more prod yml on different license plate. Even though it is not used, it it better to have both prod yml in sync right?